cinema/models.py

```diff
@@ -35,12 +40,19 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(instance: "Movie", filename: str) -> pathlib.Path:
+    ext = os.path.splitext(filename)[1]
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}{ext}"
+    return os.path.join("movies/", filename)
```

Comment:
Code Efficiency: You have defined a function `movie_image_path` but it seems you are not using it anywhere in the code. Please, remove the unused function or use it in the appropriate place.